specification/cosmos-db/data-plane/Microsoft.Tables/preview/2019-02-02/table.json

```diff
@@ -1908,6 +1932,28 @@
       "type": "string",
       "x-ms-parameter-location": "method",
       "description": "The row key of the entity."
+    },
+    "acceptJsonParameter": {
+      "description": "",
+      "name": "Accept",
+      "in": "header",
+      "type": "string",
+      "enum": [
+        "application/json"
+      ],
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "acceptXmlParameter": {
```

Comment:
Does this only accept xml? I wonder if we should update the "consumes" section to include xml too.

Reply:
The naming is meant to align with the header name, but I could see an argument for naming them to align with `consumes` and `produces` properties. In that case, I think this aligns with the `produces` side since I believe the `Accept` header indicates the acceptable response format. 

In the case where this parameter is being used, we already define both.

```json
"put": {
"tags": [
  "table"
],
"consumes": [
  "application/xml"
],
"produces": [
  "application/xml"
],
"operationId": "Table_SetAccessPolicy",
"description": "Sets stored access policies for the table that may be used with Shared Access Signatures.",
"parameters": [
  ...
  {
    "$ref": "#/parameters/acceptXmlParameter"
  }
],
```